README.md

```diff
@@ -48,28 +48,28 @@ Metis uses postgres to store a lookup table and accelerate the process of return
 
 ## About Docker Compose
 
-The current version of [Iris](https://github.com/repositive/iris-js) requires an AMQP server. The docker-compose comes preconfigured to connect to one out of the box. You'll need to run an instance yourself and attach it to the network rabbit, and to the postgres database:
+The current version of [Iris](https://github.com/repositive/iris-js) requires an AMQP server. The docker-compose comes preconfigured to connect to one out of the box. You'll need to run an instance yourself and attach it to the network rabbit, and to the postgres database. For this purpose please use the devops service:
 
```

Comment:
I'm not sure this is entirely correct. We only have a comprehensive devops service for PDX at the moment. It would be useful to have one for core platform as well, but someone needs to put that together.

Reply:
Hmm... Can we discuss this point with @Istar-Eldritch ?

Reply:
Sure. 
The concept is fine, but as the devops package is currently used for pdx you need to be aware that it won't necessarily spin up the correct services you will need.